framework/aster-frame/src/task/processor.rs

```diff
@@ -123,6 +123,10 @@ fn switch_to_task(next_task: Arc<Task>) {
     //    context and the next task context.
     // 2. The next task context is a valid task context.
     unsafe {
+        if let Some(next_user_space) = next_task.user_space() {
+            next_user_space.vm_space().activate();
+        }
```

Comment:
Actually, `VmSpace::activate` is no longer unsafe after #713 (which is quite reasonable), so maybe move it out of the unsafe block?

Reply:
Done